web/views/user_view.ex

```diff
@@ -41,4 +41,13 @@ defmodule CodeCorps.UserView do
     if user.id == current_user.id, do: user.email, else: ""
   end
   def email(_user, _conn), do: ""
+
+  @doc """
+  Returns the user's full name when both first and last name are present.
+  Returns only user's first or last name when one is missing, or an empty
+  string when both are missing
```

Comment:
I think we may want to simply return `null` if both are missing. That seems more appropriate to me here. What do you think?